### PR TITLE
Design

### PR DESCRIPTION
**Pull request by sawmon71293**
##  Update Math-magicians
- update UI
- add some operations

### CODE REVIEW

File: src/component/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
+import { useReducer } from 'react';
+import { DigitButton, ACTIONS } from './DigitButton';
+import OperationButton from './OperationButton';
+import evaluate from './Evaluate';
+
+// eslint-disable-next-line consistent-return
```
Comment (t-yanick) [inline src/component/Calculator.js]: - [ ] Great job working on this milestone. You should always avoid disabling linters as they are there to ensure clean and professional code. Rather fix the errors that are identified by the linter checks.

File: src/component/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+function reducer(state, { type, payload }) {
```
Comment (t-yanick) [inline src/component/Calculator.js]: **- [OPTIONAL ]**
You have made use of reducers which is great of which it is not a requirement for this milestone to use reducers. However, good structure of a react app that makes use of reducers always requires that you have a separate folder for reducers in which you will have your reducers and actions. It is not professional to have the reducer and actions code in your components as is the case with your project.

File: src/component/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  // eslint-disable-next-line default-case
```
Comment (t-yanick) [inline src/component/Calculator.js]: - [ ]  Great job working on this milestone. You should always avoid disabling linters as they are there to ensure clean and professional code. Rather fix the errors that are identified by the linter checks.

File: src/component/Evaluate.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+export default function evaluate({ currentOperand, previousOperand, operation }) {
+  const prev = parseFloat(previousOperand);
+  const current = parseFloat(currentOperand);
+  if (Number.isNaN(prev) || Number.isNaN(current)) return '';
+  let computation = '';
+  // eslint-disable-next-line default-case
```
Comment (t-yanick) [inline src/component/Evaluate.js]: - [ ]  Great job working on this milestone. You should always avoid disabling linters as they are there to ensure clean and professional code. Rather fix the errors that are identified by the linter checks.

File: src/component/Calculator.js [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+      <DigitButton
+        digit="3"
+        dispatch={dispatch}
+      />
+      <OperationButton
+        operation="+"
+        dispatch={dispatch}
+      />
+      <DigitButton
+        digit="0"
```
Comment (t-yanick) [inline src/component/Calculator.js:97]: - [ ] As per the design, when the calculator is ON and not is use, the digit 0 is displayed on the screen by default. But this is not the case with your design at the moment. Kindly fix this issue.
- Also make the displayed digit to be bold as yours is not currently bold.
**Given design**
image

**Your design**
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @sawmon71293 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  All CRA boilerplate deleted
:heavy_check_mark: Component directory present, having Calculator.js file which is functional

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Capitalising the first letters of your commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — APPROVED** [review]
Hi @sawmon71293 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**